README.md

```diff
@@ -5,6 +5,9 @@ This service allows unauthenticated users to claim Child Benefit by answering th
 
 The service calls the Bank Account Reputation service’s [Validate bank details endpoint](https://github.com/hmrc/bank-account-reputation/blob/main/docs/eiscd/v3/validateBankDetails.md), but otherwise does not integrate with any other services beyond those ubiquitous to the platform (audit, contact, tracking consent etc.)
 
+### How to run the service
+You can run the service using service manager with profile `CLAIM_CHILD_BENEFIT_ALL` `CLAIM_CHILD_BENEFIT_FRONTEND` or locally with `sbt "run 11303"`
+
```

Comment:
Perhaps add a few lines describing how to log in - providing the starting URL for the auth wizard, some NINOs for sanity checking the service, and point to the stub for the full documentation.